ios/EmbedReactNative.swift

```diff
@@ -311,7 +333,7 @@ class EmbedReactNative: NSObject {
                   ]
                 )
                 return
-              case .transactionCreated(let transactionID, let status, let paymentMethodID):
+              case .transactionCreated(let transactionID, let status, let paymentMethodID, let approvalUrl):
```

Comment:
This is needed by the latest update otherwise the build fails https://github.com/gr4vy/gr4vy-ios/pull/10